examples/ui5-js-app/webapp/test/e2e/exec.test.js

```diff
@@ -0,0 +1,181 @@
+const Main = require("./pageObjects/Main")
+const Other = require("./pageObjects/Other")
+const marky = require("marky")
+const { wdi5 } = require("wdio-ui5-service")
+
+describe("ui5 eval on control", () => {
+    before(async () => {
+        await Main.open()
+    })
+
+    it("should have the right title", async () => {
+        const title = await browser.getTitle()
+        expect(title).toEqual("Sample UI5 Application")
+    })
+
+
+    it("should be able to propagate a browserside error", async () => {
+        //Log Output during this test should be 3 times: [wdi5] call of exec failed because of: TypeError: this.getTex is not a function
```

Comment:
optional enhancement: you can validate error messages on the console in the Node.js scope with the help of `sinon`.  
See https://github.com/ui5-community/wdi5/blob/16d69841b7c5d597c6d9712b3c64207cca750ebc/examples/ui5-js-app/webapp/test/e2e/error-logging.test.js#L26